notebooks/tr/agents.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<ul><li>LLM'e (elelem)</li><li>Bu çok otomatik çeviri gibi, çeviri yaparken kendimiz çevirelim ya da akıcılığa özen gösterelim yoksa okuyucular kaçar: Karşılaştığımız zorluk ise:&nbsp;<strong>Bu parametreleri kullanıcı sorgusuna göre nasıl dinamik olarak ayarlayabiliriz?</strong></li><li>RAG'in*</li><li>RAG'de sık karşılaşılan bir sorun, kullanıcı sorgusuna verilen cevabın hangi belgeden geldiğinin bulunmamasıdır.</li><li>İlk olarak üzerinde RAG uygulamak*</li><li>bilgi tabanı yerine veriseti</li><li>Hugging Face* kütüphanesinin</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>